spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -141,4 +141,46 @@ void testTransformUpdateColumnType() {
     Assertions.assertTrue(
         "string".equalsIgnoreCase(gravitinoUpdateColumnType.getNewDataType().simpleString()));
   }
+
+  @Test
+  void testTransformUpdateColumnPosition() {
+    TableChange.ColumnPosition first = TableChange.ColumnPosition.first();
```

Comment:
could you add the test for default position?

Reply:
got it, and i will add the default position case.

Reply:
updated.

Reply:
seems not upated? I couldn't find the code to test default position